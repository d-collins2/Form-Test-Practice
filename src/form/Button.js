import React from 'react';
import { Button } from 'react-bootstrap'

const FormButton = ({text}) => {
    return (
        <div className="text-right">
            <Button className="btn btn-primary" type="submit"> {text} </Button>
        </div>
    )
}

FormButton.defaultProps = {
    options: []
}

export default FormButton;
