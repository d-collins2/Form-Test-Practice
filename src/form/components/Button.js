import React from 'react';
import { Button } from 'react-bootstrap'

const FormButton = ({label}) => {
    return (
        <div className="text-right">
            <Button className="btn btn-primary" type="submit">{label}</Button>
        </div>
    )
}

FormButton.defaultProps = {
    options: []
}

export default FormButton;
