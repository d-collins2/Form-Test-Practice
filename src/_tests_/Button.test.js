import renderer from 'react-test-renderer';
import Button from '../form/Button';
import React from 'react';
import { shallow } from 'enzyme';

describe('<Button />', function () {
    it('Should render correctly', function () {
        const component = renderer.create(<Button />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
})
