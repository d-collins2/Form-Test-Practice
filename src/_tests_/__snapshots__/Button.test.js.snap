// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Should render correctly 1`] = `
<div
  className="text-right"
>
  <button
    className="btn btn-primary btn btn-primary"
    disabled={false}
    type="submit"
  >
     
     
  </button>
</div>
`;
